test/jdk/java/awt/print/PrinterJob/ValidatePage/ValidatePage.java

```diff
@@ -213,8 +209,9 @@ public ValidatePage() {
                 myPageFormat.setPaper(p1);
                 displayPageFormatAttributes();
             } catch (NumberFormatException nfe) {
-                PassFailJFrame.forceFail( "Test Failed");
-                nfe.printStackTrace();
+                JOptionPane.showMessageDialog(null,
+                        "NumberFormatException occured", "Error",
+                        JOptionPane.ERROR_MESSAGE);
```

Comment:
```suggestion
                JOptionPane.showMessageDialog(ValidatePage.this,
                        "NumberFormatException occurred", "Error",
                        JOptionPane.ERROR_MESSAGE);
```
You may provide more information to the user, at the very least the message from the exception itself. Add a comment that the tester should correct the error and try again.